docs/modules/ROOT/pages/libraries.adoc

```diff
@@ -784,3 +784,99 @@ DS.Util.mapToObject(payload, function(x) x.language.name, function(v) v.language
 };
 ------------------------
 
+## Regex
+
+The regular expressions library based on the https://github.com/google/re2j[Google RE2/J] implementation.
```

Comment:
Missing "is"; also, I think we can just talk about our regular expressions library and only link to the RE2/RE2j docs as needed (pretty much only when defining what syntax we support, I think?)